GATEWAY/src/pages/applications/applications.ts

```diff
@@ -28,7 +28,17 @@ export class ApplicationsPage {
               public modalCtrl: ModalController,
               private mqttClient: MqttClient,
               private tilesApi: TilesApi,
-              private storage: Storage) {}
+              private storage: Storage,
+              private plt: Platform,
+              private alertCtrl: AlertController) {
+
+    if (this.plt.is('android')) {
```

Comment:
I think the test should be in the check in ble-service that checks if ble is enabled and then check for location if ble was enables and the platform is android